### ISSUE TITLE
vs code crashes all the time on Ubuntu 20.04LTS

### ISSUE THREAD

**Issue by Andonerei** [microsoft/vscode#137610]
Issue Type: <b>Bug</b>

There is no exact way to reproduce. I just can run code, leave it untouched for a while, for example, i go to get some coffee and when i'm back vs code is already gone.

This is reproduced quite often, several times an hour.

VS Code version: Code 1.62.3 (ccbaa2d27e38e5afa3e5c21c1c7bef4657064247, 2021-11-17T08:00:36.721Z)
OS version: Linux x64 5.11.0-40-generic snap
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8265U CPU @ 1.60GHz (8 x 1800)|
|GPU Status|2d_canvas: unavailable_software<br>gpu_compositing: disabled_software<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: disabled_off<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: unavailable_software<br>webgl2: unavailable_software|
|Load (avg)|0, 0, 0|
|Memory (System)|7.42GB (0.94GB free)|
|Process Argv|--no-sandbox --force-user-env --unity-launch --crash-reporter-id 99b476e2-2ec6-42ec-a0d8-6898f8f93b25|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (7)</summary>

Extension|Author (truncated)|Version
---|---|---
add-reference|adr|1.0.2
nugetpackagemanagergui|ali|1.1.9
csharpextensions|jch|1.3.1
dotnet-core-essentials|Kis|0.0.8
csharpextensions|kre|1.5.2
csharp|ms-|1.23.16
vscode-nuget-gallery|pat|0.0.24

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonvspyt551:30345470
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30384386
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
dockerwalkthru:30377721
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsqsis200cf:30402084
vsaa593:30376534
vssld246:30401890
vscexrecpromptt3:30400063

```

</details>

<!-- generated by issue reporter --
[vscode.crash.zip](https://github.com/microsoft/vscode/files/7576318/vscode.crash.zip)
>

**Comment by Andonerei** [comment]
attaching the dump [vscode.crash.zip](https://github.com/microsoft/vscode/files/7576346/vscode.crash.zip)

**Comment by mxrachel** [comment]
does it still happen if you disable all extensions `code --disable-extensions`?

**Comment by Andonerei** [comment]
yes
i attach the new dump with extensions disabled
[vscode.crash.ext-disabled.zip](https://github.com/microsoft/vscode/files/7576618/vscode.crash.ext-disabled.zip)

**Comment by Andonerei** [comment]
The issue is still relevant for the newer version

Version: 1.63.2
Commit: 899d46d82c4c95423fb7e10e68eba52050e30ba3
Date: 2021-12-15T09:39:46.686Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Linux x64 5.13.0-27-generic snap

**Comment by deepak1556** [comment]
The issue should be addressed in latest stable `1.73`, please check if it still persists.
